vnext/src/Libraries/Components/Button.windows.js

```diff
@@ -335,7 +335,67 @@ class Button extends React.Component<
     //  Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
     // Windows]
     // [Windows
-    if (Platform.OS === 'windows') {
+    if (Platform.OS === 'windows' && color) {
+      return (
+        <Touchable
+          accessibilityLabel={accessibilityLabel}
+          accessibilityRole="button"
+          accessibilityState={accessibilityState}
+          hasTVPreferredFocus={hasTVPreferredFocus}
+          nextFocusDown={nextFocusDown}
+          nextFocusForward={nextFocusForward}
+          nextFocusLeft={nextFocusLeft}
+          nextFocusRight={nextFocusRight}
+          nextFocusUp={nextFocusUp}
+          testID={testID}
+          disabled={disabled}
+          onPress={onPress}
+          tabIndex={tabIndex}
+          touchSoundDisabled={touchSoundDisabled}
+          underlayColor="white"
+          onShowUnderlay={() => {
+            this.setState({pressed: true});
+          }}
+          onHideUnderlay={() => {
+            this.setState({pressed: false});
+          }}
+          onMouseEnter={() => {
+            if (!disabled) this.setState({hover: true});
+          }}
+          onMouseLeave={() => {
+            if (!disabled) this.setState({hover: false});
+          }}>
+          <View
+            style={
+              this.state.pressed
+                ? [
+                    buttonStyles,
+                    {
+                      borderColor: PlatformColor('ButtonBorderBrushPressed'),
```

Comment:
Where do these colors come from/ why are they used with color set?

Reply:
Same for below background color

Reply:
ButtonBorderBrushPressed is a XAML brush used for the color the XAML Button border in the pressed state. In the ideal scenario for the colored button, the border colors would be deepened versions of the input color itself. But, at present in RN there isn't a way to formulate additional colors from a ColorValue variable and so to generate the shadow effect we see with the XAML Button we are using the border colors of the default button which end up being gray tones.